### ISSUE TITLE
How to install this application?

### ISSUE THREAD

**Issue by ghost** [linuxflyos/flyos-campus-network#4]
Does the application has deb package?
or flatpak package?

**Comment by xiayesuifeng** [comment]
No, my computer does not have a debian-based distribution. But you can build and install it based on the readme. As long as there is a dependency, the build is simple
